shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1310,6 +1311,38 @@ fn test_passthrough() {
     prepared_statements::test(&connection);
 }
 
+#[test]
+#[serial]
+fn test_passthrough_tls() {
+    let _compose = DockerCompose::new("examples/cassandra-passthrough-tls/docker-compose.yml")
+        .wait_for_n_t("Startup complete", 1, 90);
+
+    let shotover_manager =
+        ShotoverManager::from_topology_file("examples/cassandra-passthrough-tls/topology.yaml");
+
+    let tls_config = TlsConfig {
+        certificate_authority_path: "examples/cassandra-passthrough-tls/certs/db.crt".into(),
+        certificate_path: "examples/cassandra-passthrough-tls/certs/cadb.pem".into(),
+        private_key_path: "examples/cassandra-passthrough-tls/certs/cadb.key".into(),
+    };
+
+    let connection = shotover_manager.cassandra_connection_tls(
+        "127.0.0.1",
+        9043,
+        tls_config,
+        "cassandra",
+        "cassandra",
+    );
+
+    keyspace::test(&connection);
```

Comment:
Can we run a simple test case directly against cassandra to make sure that cassandra_connection_tls can connect to both cassandra and shotover?
This would protect against a possible case where both shotover and cassandra_connection_tls were doing something slightly wrong compared to actual cassandra.
I think `table::test` would be fast enough for this use case.

Reply:
Decided to just run a simple query against the `system.local` table, rather than implement teardown code for the `table::test` suite.

Reply:
Decided to just run a simple query against the `system.local` table, rather than implement teardown code for the `table::test` suite.

Reply:
nice!